src/nodejs_bindings.cpp

```diff
@@ -44,8 +44,8 @@ NAN_METHOD(Annotator::New)
     }
     else
     {
-        auto init = Nan::New(constructor());
-        info.GetReturnValue().Set(init->NewInstance());
+        return Nan::ThrowTypeError(
+            "Cannot call constructor as function, you need to use 'new' keyword");
```

Comment:
Why the change here? The `Lookup::New` function still supports both `new T()` as well as `T()`

https://github.com/mapbox/route-annotator/pull/5/files#diff-e03b7689579005881fd2f324670dbfbfR22

Either allow it both the Annotator and the Lookup Js type or neither.